dockers/docker-database/database_config.json.j2

```diff
@@ -102,23 +111,23 @@
         "DPU_APPL_DB" : {
             "id" : 15,
             "separator": ":",
-            "instance" : "redis",
+            "instance" : {% if include_remote_db%} "remote_redis" {% else %} "redis" {% endif %},
```

Comment:
@Pterosaur which design doc are you following? I am not sure, if the DPU_<Table Name> will be hosted on the NPU side or DPU side?

Reply:
Add the design doc to the description and that has been reviewed by community.